index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>Pizza</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link href="css/reset.css" rel="stylesheet">
+	<link href="css/main.css" rel="stylesheet">
+
+	<script src="js/main.js"></script>
+</head>
+<body onload="digitalWatch()">
+	<header>
+		<div class="header-wrapper">
+			<div id="digital_watch" class="watch-wrapper">
+			</div>
+		</div>
+		<div class="header-wrapper">
+			<a href="#" class="col-wrapper">
+				<div class="hvr-wobble-top"></div>
+				<p>Pizza Manager</p>
+			</a>
+		</div>
+		<div class="header-wrapper">
+			<div class="col-wrapper col-wrapper-grid">
+				<button>SING IN</button>
+				<button>SING UP</button>
+			</div>
+		</div>
+	</header>
+
+	<content>
```

Comment:
Avoid using `<content>` element. This feature has been removed from the Web standards.
Semantic element `<main>` is much better choice